udemy_enroller/cli.py

```diff
@@ -23,31 +23,49 @@ def enable_debug_logging() -> None:
 
 
 def determine_if_scraper_enabled(
-    tutorialbar_enabled: bool,
-    discudemy_enabled: bool,
-    coursevania_enabled: bool,
-) -> Tuple[bool, bool, bool]:
+        freebiesglobal_enabled: bool,
+        comidoc_enabled: bool,
```

Comment:
Did you test out comidoc? 
I think we removed it a while back as it was unstable (i.e always changing)

Reply:
Let's not add back comidoc, it's extremely unstable.

Reply:
I agree.

Reply:
Removed